### ISSUE TITLE
Android 14 Crash com.hyphenate.chat.core.EMAdvanceDebugManager.h java.lang.SecurityException

### ISSUE THREAD

**Issue by utkudenis** [AgoraIO/Agora-Chat-API-Examples#79]
Hi Agora, 

I cannot find agora flutter chat_sdk repo on the github so ı opened  issue to  this repo. 

We are getting crashes on the production. 
sdk version 1.1.1
flutter version 3.13.9 
Are you aware of the problem on the android 14 ? 
should I update package version to 1.2.0 to solve problem. ?

Crash Details 
Stack trace
Sample attributes:
[samsung dm1q (Galaxy S23)]
[Android 14 (SDK 34)]
`Exception java.lang.SecurityException:
  at android.os.Parcel.createExceptionOrNull (Parcel.java:3069)
  at android.os.Parcel.createException (Parcel.java:3053)
  at android.os.Parcel.readException (Parcel.java:3036)
  at android.os.Parcel.readException (Parcel.java:2978)
  at android.app.IActivityManager$Stub$Proxy.registerReceiverWithFeature (IActivityManager.java:6137)
  at android.app.ContextImpl.registerReceiverInternal (ContextImpl.java:1913)
  at android.app.ContextImpl.registerReceiver (ContextImpl.java:1853)
  at android.app.ContextImpl.registerReceiver (ContextImpl.java:1841)
  at android.content.ContextWrapper.registerReceiver (ContextWrapper.java:772)
  at com.hyphenate.chat.core.EMAdvanceDebugManager.h
  at com.hyphenate.chat.core.EMAdvanceDebugManager.a
  at com.hyphenate.chat.EMClient.onNewLogin
  at com.hyphenate.chat.EMClient$8.run
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1145)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:644)
  at java.lang.Thread.run (Thread.java:1012)
Caused by android.os.RemoteException: Remote stack trace:
  at com.android.server.am.ActivityManagerService.registerReceiverWithFeature (ActivityManagerService.java:16580)
  at android.app.IActivityManager$Stub.onTransact$registerReceiverWithFeature$ (IActivityManager.java:11530)
  at android.app.IActivityManager$Stub.onTransact (IActivityManager.java:2928)
  at com.android.server.am.ActivityManagerService.onTransact (ActivityManagerService.java:3195)
  at android.os.Binder.execTransactInternal (Binder.java:1375)`

**Comment by dujiepeng** [comment]
@utkudenis 
Yes. Version 1.2.0 is compatible with android 14, along with bug fixes and new features.

https://pub.dev/packages/agora_chat_sdk/changelog
